.github/workflows/test.yaml

```diff
@@ -10,8 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8]
-        django-version: [2.2, '3.0', 3.1]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django-version: ['2.2', '3.1', '3.2']
```

Comment:
Why are single quotes needed around the django versions? But not the python and postgres versions?

Reply:
```suggestion
        django-version: [2.2, 3.1, 3.2]
```

Reply:
For some yet unknown reason the Github CI pipeline fails when using the unquoted Django versions (_tested in https://github.com/warpnet/django-nacl-fields/commit/e389f1737b1fa8711b656ad9ff222b84db847865_). Instead of installing Django version `3.1`, it tries to install version `3`.